.pre-commit-config.yaml

```diff
@@ -103,20 +104,23 @@ repos:
     hooks:
       - id: clang-format
         name: C/GLib Format
+        alias: c-glib-format
         files: >-
           ^c_glib/
   - repo: https://github.com/pre-commit/mirrors-clang-format
     rev: v14.0.6
     hooks:
       - id: clang-format
         name: MATLAB (C++) Format
+        alias: matlab-format
         files: >-
           ^matlab/src/cpp/
   - repo: https://github.com/pre-commit/mirrors-clang-format
     rev: v14.0.6
     hooks:
       - id: clang-format
         name: Python (C++) Format
+        alias: python-format
```

Comment:
This should probably be better named `python-cpp-format` otherwise it seems like it is formatting the Python code, which is done at the flake8 stage.

Probably the same for other languages like R or Matlab as this is formatting the C++ code for those.

Reply:
@raulcd Thank you for your comments. I added `-cpp`. @kou What do you think `c-glib-cpp-format` and `c-glib-format`?

Reply:
It makes sense. `c-glib-cpp-format` is better because we may add `c-glib-ruby-format` for `c_glib/test/` later.